app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -1,6 +1,12 @@
 # Copyright © Magento, Inc. All rights reserved.
 # See COPYING.txt for license details.
 
+type StoreConfig @doc(description: "The type contains information about a store config") {
```

Comment:
`StoreConfig` is described in the StoreGraphQl schema file. Is it necessary to repeat it?

Reply:
Removed description for `StoreConfig`